### ISSUE TITLE
Support Chrome Extension UI like Settings, Popup, etc.

### ISSUE THREAD

**Issue by PramodValavala-MSFT** [microsoft/accessibility-insights-web#5714]
## Is your feature request related to a problem? Please describe.

Currently, it is not possible to run Accessibility Insights on a chrome extension page like a custom settings page. When trying to do so, one gets this message

![image](https://user-images.githubusercontent.com/43602528/177989026-7f4bcb66-24ea-4eeb-a3f9-81a9a70312dc.png)

## Describe the desired outcome

The extension should work with chrome extension pages

## Describe alternatives you've considered

The one solution seems to be using Accessibility Insights for Windows

**Comment by dbjorge** [comment]
Hi @PramodValavala-MSFT,

Unfortunately, this is a technical restriction enforced by the browser as an intentional security boundary, and isn't within our control to change. In general, browser extensions are not allowed to inject content scripts into other browser extensions' pages, which means we can't run any accessibility scans/etc there. The error message that Edge and Chrome give us if we were to try looks like `Error: Cannot access a chrome-extension:// URL of different extension`.

As a partial solution, you can run the equivalent of Accessibility Insights' Automated Checks by using `axe-core` (the accessibility scanning engine Accessibility Insights is based on) from the context of an automated end-to-end test of your extension. This is how we run automated checks against Accessibility Insights for Web itself. We recommend using [Playwright](https://playwright.dev/) for this: It supports [working with Chrome Extensions](https://playwright.dev/docs/chrome-extensions) and has good ecosystem support for [accessibility testing with `axe-core`](https://playwright.dev/docs/next/accessibility-testing).

**Comment by dbjorge** [comment]
We think that if we were to someday support this, it would probably be through a mechanism similar to what we someday hope to use for scanning of Electron apps. I've updated the existing issue tracking that, #558, to make it clear that this request would fall inside that scope; I'm going to close this issue as a duplicate, but feel free to post any followup discussion/upvotes/etc over in #558.

**Comment by PramodValavala-MSFT** [comment]
@dbjorge Thanks for the insights! I'll give the approach you've mentioned a try.

I understand the rationale behind closing this. The out-of-process design does make sense for this and the other scenarios you've mentioned there. 👍🏽
